exporters/metric/cortex/example/docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: "3"
+
+services:
+  app:
+    image: golang:1.14-alpine
+    volumes:
+      - .:/app
+    working_dir: /app
+    command: go run main.go
+  cortex:
+    image: quay.io/cortexproject/cortex:v1.3.0-rc.1
```

Comment:
Is there a need to use this release-candidate instead of a stable release?

Reply:
There's no stable "latest" or comparable version. That was the latest versioned number, the rest are master-<commit-hash>.

Reply:
Gotcha. Didn't mean to imply we should track `latest` here, but instead something like `1.2.0` (sorry cannot figure out how to link it):

![20200818_100817](https://user-images.githubusercontent.com/5543599/90543656-e08e5b00-e13a-11ea-94da-f255aa7362d9.png)

It's not a huge deal. I was more so curious if that release candidate contained needed functionality.

Reply:
Makes sense, I'll double check but I don't think it contains any changes we need. If using 1.2.0 would be more stable, I'll switch to that.

Reply:
Actually, we do need 1.3 to use the most recent configuration.